test/integration/scheduler_perf/main_test.go

```diff
@@ -17,11 +17,38 @@ limitations under the License.
 package benchmark
 
 import (
+	"flag"
+	"fmt"
+	"os"
 	"testing"
 
+	"github.com/spf13/pflag"
+	logsapi "k8s.io/component-base/logs/api/v1"
+	"k8s.io/klog/v2/ktesting"
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
 func TestMain(m *testing.M) {
+	// Run with -v=2, this is the default log level in production.
+	ktesting.DefaultConfig = ktesting.NewConfig(ktesting.Verbosity(2))
+	ktesting.DefaultConfig.AddFlags(flag.CommandLine)
+
+	c := logsapi.NewLoggingConfiguration()
+
+	// component-base only supports pflag at the moment.
+	var fs pflag.FlagSet
+	logsapi.AddFlags(c, &fs)
+	// Not ideal. https://github.com/spf13/pflag/pull/330 would be better.
+	fs.VisitAll(func(f *pflag.Flag) {
+		if flag.CommandLine.Lookup(f.Name) == nil {
+			flag.CommandLine.Var(f.Value, f.Name, f.Usage)
+		}
+	})
+	flag.Parse()
```

Comment:
It's coming into logsapi, but isn't there yet: https://github.com/kubernetes/kubernetes/pull/114731

Reply:
@kerthcet please add `TODO(#114731)` to make sure to remove this code

Reply:
Looks like that is getting reviewed sooner than I thought. I still need an API approver, though.

It might make sense to wait for it - up to you. If not, then I'll update it once it both PRs are merged.

Reply:
Let's wait

Reply:
@kerthcet: https://github.com/kubernetes/kubernetes/pull/114731 got merged, please replace this section with `logsapi.AddGoFlags(flag.CommandLine)`.

Reply:
When I added the `logsapi.AddGoFlags(flag.CommandLine)`, error happens, see below. Let me cover this commit in another PR and only focus on fixing CI error in this one.
```
/var/folders/c_/7mkl6xtj6qz3dpn1ggg4d2yw0000gn/T/go-build3763591496/b001/scheduler_perf.test flag redefined: log-flush-frequency 
```

Reply:
Ah, that's because of the `if flag.CommandLine.Lookup(f.Name) == nil {` check that isn't in `AddGoFlags`. As you said elsewhere, let's deal with this in a separate PR. I can also take over that part if you want.

Reply:
That would be great Patrick, thanks.